packages/flutter/test/gestures/force_press_test.dart

```diff
@@ -0,0 +1,230 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
you need some tests to catch the issues i raised
you need some tests to test the details object

Reply:
I don't see any tests for details objects for other gesture detectors?